test/unitTests/build/buildExecutor.test.ts

```diff
@@ -245,15 +242,14 @@ describe('BuildExecutor', () => {
         it('Build Cancelled', (done) => {
             mockExecuteDocfx(0, 0);
             killProcessTreeFuncCalled = false;
-            let buildPromise: Promise<BuildResult>;
 
             subscription = eventStream.subscribe((event) => {
                 if (event.type === EventType.DocfxBuildStarted) {
                     setTimeout(async () => {
                         let buildResult: BuildResult;
                         try {
                             await buildExecutor.cancelBuild();
-                            buildResult = await buildPromise;
+                            buildResult = await buildExecutor.RunBuild('fakedCorrelationId', fakedBuildInput, 'faked-build-token');

```

Comment:
This change will make `buildExecutor.RunBuild` never get called, so the test timeout.

Reply:
Undid and suppressed in-line